modules/administration-guide/pages/limiting-the-number-of-workspaces-that-a-user-can-keep.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: PROCEDURE
+:navtitle: Limiting the number of workspaces that a user can keep
+:description: Limiting the number of workspaces that a user can keep
+:keywords: administration guide, number, workspaces
+:page-aliases: configuring-the-number-of-workspaces-that-a-user-can-create.adoc
+
+[id="limiting-the-number-of-workspaces-that-a-user-can-keep_{context}"]
+= Limiting the number of workspaces that a user can keep 
+
+{prod-short} enables users to keep several workspaces in their dashboards.
+
+By default, users can keep an unlimited number of workspaces.
+
+You can set a limit on the number of workspaces that a user can keep in the dashboard.
```

Comment:
"As an administrator, you can limit..."
Does it make sense to mention the role of the person performing this task?

Reply:
Thanks for the suggestions, Rolfe.

_Limiting this number <describe benefit>._ Thank you, lack of this sentence has been nagging me too. I need to ask the SME to phrase it.

:slightly_smiling_face: _Lead with the main point._ I completely agree with you on this point in principle. In this case, the main point requires some context setting, so what I'm trying to do here is first set the necessary context for the main point.

:slightly_smiling_face: Vale forced me to change it to "_several_". I initially phrased it as "_a number of_" (or that's what I remember) and Vale treated that as an error.

:slightly_smiling_face: From my personal experience, I find that some (not implying you in particular here, some indeed) technical writers take the notion of anthropomorphism to an extreme and apply it to words that aren't.
Have a look at the dictionary examples for "_enable_":
https://www.merriam-webster.com/dictionary/enable
_`training that enables` people to earn a living
to make possible, practical, or easy a `deal that would enable` passage of a new law
to cause to operate `software that enables` the keyboard
to give legal power, capacity, or sanction to a `law enabling admission` of a state_
https://dictionary.cambridge.org/us/dictionary/english/enable
_This `software will enable` us to track orders more efficiently.
Recent `technology has enabled` the development of cheaper digital radios._
I guess that's enough for me to use the word "_enable_" from now on without looking back at the idea of anthropomorphism.
But you get the idea, let's say I start treating the word "_enable_" as an anthropomorphism, what will happen? I will replace it with maybe "_makes it possible_", but then again, a person can make something possible too, so will that too become an anthropomorphism on the basis that it may, among other things, be applicable to a person?

:slightly_smiling_face: It's in the admin guide, so I guess "_As an administrator, you can limit..._" is redundant (and if I apply it here, then it'll have to be used virtually everywhere else throughout the admin guide).

:slightly_smiling_face: I'm not happy with my current wording, but in your suggestion the connotation is altered:
"_By default, users can keep an unlimited number of workspaces._" (users keep an unlimited number of, which is a result of user activity)
versus
"_You can limit the number of workspaces each user can have in their dashboard. By default, this number is unlimited._" (the number that you can limit is unilimited by default, which leans toward a setting that admin has to deal with)

"_The product is apparent from the context._" Thank you, I'm making another suggestion below.